config/vufind/ComposedDriver.ini

```diff
@@ -0,0 +1,36 @@
+[General]
+; Default driver for all methods
+;main_driver = "driver1"
+
+; (Optional) the path to the driver configurations relative to VuFind config
+; dir -- omit to locate those in VuFind config dir -- it also could be an
+; absolute path
+;drivers_config_path = private
+
+; Definition of all the used drivers in the form <DriverName> = <DriverClass>
+; The name of config file for a driver needs te be the same <DriverName>
+[Drivers]
+;driver1 = PAIA
+;driver2 = DAIA
+;driver3 = Demo
+
+; For each driver method it can be specified which drivers should be used and how they can be composed.
+; The head needs to be [<driver method name>]. main_driver specifies the driver that is primarily used.
```

Comment:
I just noticed this one: I think we use `section` instead of `head`, so I'd change this for consistency.

Reply:
done